cert/spec/runner_spec.rb

```diff
@@ -95,7 +101,7 @@
       end
 
       let(:generate) do
-        options = { output_path: temp, filename: filename }
+        options = { output_path: temp, filename: filename, keychain_path: "." }
```

Comment:
This is workaround for tests failing on Ubuntu machine. Default argument value for `keychain_path` (`~/Library/Keychains/login.keychain`) is not valid on Ubuntu.

I am not sure if and how cert works on Ubuntu and workarounds keychain? I think concept of macOS keychain is not available on Ubuntu (`security` command doesn't work)?

Reply:
We probably don't want this as a work around just for Ubuntu tests because this means that if anybody tries to run `cert` in Ubuntu without passing a `keychain_path` of `"."`, it will probably fail in real life 😬

We might want to do a check for macOS somewhere instead? I think @janpio might have a better suggestion here since he has more of the non-macOS experience 😇

Reply:
I am open for suggestions. However, `keychain_path` has default value and is not optional argument. So it seems to me that `cert` hasn't been working properly on Ubuntu anyway.

It is true that this change pushed `Cert.config[:keychain_path]` up in code path which triggered exception earlier. So it might break some of existing code.

Is there a way to do `Cert.config[:keychain_path]` so that it doesn't raise error if `keychain_path` is not valid? Or check if it is valid somehow?

Reply:
`cert` is probably broken on Ubuntu outside of the tests if there is no special handling for "keychain doesn't actually exist on non-macOS" yet. So I am good with working with `.` here, although a comment might not be a bad idea. One day we might want to get it to work there, and this might be a valuable pointer.

Reply:
What about skipping the tests `unless FastlaneCore::Helper.mac?`?

Reply:
The tests are actually passing, so no need to skip. The test is not _really_ testing all the functionality. The real solution here is to go in deeper and make sure all the keychain stuff is limited to macOS - but that is quite a lot of work and outside the scope of this PR.